Src/Plugins/Security/SEC102.ReviewPotentiallySensitiveData.json

```diff
@@ -22,6 +24,13 @@
           "Name": "ReviewPotentiallySensitiveData/SocialSecurityNumber",
           "ContentsRegex": "\\b(?P<refine>(?P<secret>[0-9]{3}-[0-9]{2}-[0-9]{4}))\\b",
           "MessageArguments": { "dataKind": "social security number" }
+        },
+        {
+          "Id": "SEC102/003",
```

Comment:
>Id [](http://example.com/codeflow?start=11&length=2)

Note that this rule picks up some default information from the root location, such as the level of "note"